go/engine/box_audit_retry_background.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2018 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package engine
+
+import (
+	"sync"
+	"time"
+
+	"github.com/keybase/client/go/libkb"
+)
+
+var BoxAuditRetryBackgroundSettings = BackgroundTaskSettings{
+	Start:        2 * time.Minute,
+	StartStagger: 1 * time.Minute,
+	WakeUp:       1 * time.Minute,
+	Interval:     7 * time.Hour,
```

Comment:
This might be too aggressive.  It would be nice if we can server control this for a bit, is that possible?

Reply:
This is for the failed audit retry queue, so it's usually a noop. Server controls feature flag so we can make them all noops as well.